azure-devops/run-build.yml

```diff
@@ -19,12 +19,52 @@ jobs:
       displayName: 'Setup TMP Directory'
     - checkout: self
       clean: true
-      submodules: true
+      submodules: false
+    - task: PowerShell@2
+      displayName: 'Get submodule SHAs'
+      timeoutInMinutes: 1
+      inputs:
+        targetType: inline
+        script: |
+          cd $(Build.SourcesDirectory)
+          $pattern = "[^a-zA-Z0-9]"
+          $llvmSHA = git submodule status llvm-project | %{($_ -split "\s+")[0]}
+          $llvmSHA = $llvmSHA -replace $pattern,""
+          Write-Host "##vso[task.setvariable variable=llvmSHA;]$llvmSHA"
+          $vcpkgSHA = git submodule status vcpkg | %{($_ -split "\s+")[0]}
+          $vcpkgSHA = $vcpkgSHA -replace $pattern,""
+          Write-Host "##vso[task.setvariable variable=vcpkgSHA;]$vcpkgSHA"
+    - script: |
+        cd $(Build.SourcesDirectory)
+        if not exist "llvm-project" (
+          mkdir llvm-project
+        )
+        cd llvm-project
+        git init
+        git remote add llvm https://github.com/llvm/llvm-project
+        git config --local extensions.partialClone llvm
+        git fetch --filter=tree:0 --depth=1 -v llvm $(llvmSHA)

```

Comment:
This passes `-v` for verbosity, but the `git fetch` command for `vcpkg` below doesn't. Is this intentional, or is this a relic of development?

Reply:
It's a relic.